templates/shared/runtime/configure-clocksource.service

```diff
@@ -1,5 +1,8 @@
 [Unit]
 Description=Configure kernel clocksource
+# the script needs to use IMDS, so wait for the network to be up
+Wants=network-online.target
+After=network-online.target
 
 [Service]
```

Comment:
not sure what behavior we've seen historically, but should we also set it to restart on failure?